frontend/src/components/pod/__snapshots__/PodDetails.stories.storyshot

```diff
@@ -5471,9 +5471,7 @@ exports[`Storyshots Pod/PodDetailsView Successful 1`] = `
                 >
                   <span
                     class="MuiTypography-root MuiTypography-body1 css-1qti38g-MuiTypography-root"
-                  >
-                    0.0.0.1
```

Comment:
Seems odd this IP address isn't being rendered anymore?

Reply:
It's just because I did modify one of the mock pods to remove any IPs from it. So this is expected, even though I understand the confusion.